src/Resources/config/doctrine/model/Account.orm.xml

```diff
@@ -0,0 +1,25 @@
+<?xml version="1.0" encoding="UTF-8"?>
+
+<doctrine-mapping xmlns="http://doctrine-project.org/schemas/orm/doctrine-mapping"
+                  xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+                  xsi:schemaLocation="http://doctrine-project.org/schemas/orm/doctrine-mapping
+                                      http://doctrine-project.org/schemas/orm/doctrine-mapping.xsd">
+
+    <entity name="Setono\SyliusCriteoPlugin\Model\Account" table="setono_sylius_criteo_account">
+
+        <indexes>
+            <index columns="enabled" name="idx_enabled"/>
+        </indexes>
+
+        <id name="id" type="integer">
+            <generator strategy="AUTO"/>
+        </id>
+
+        <field name="accountId" type="string" unique="true" />
+        <field name="enabled" column="enabled" type="boolean" />
+
+        <one-to-one field="channel" target-entity="Sylius\Component\Channel\Model\ChannelInterface" />
```

Comment:
Der mangler forresten nok en [join column](https://www.doctrine-project.org/projects/doctrine-orm/en/2.6/reference/association-mapping.html#one-to-one-unidirectional) her, som der kan sættes `nullable=false` på